### PR TITLE
Feature/#242 valid anagram

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

**Review by ryujumoto — COMMENTED** [review]
Can you solve this with better time complexity?

**Comment by Rwatana** [comment]
I solve it by one sentence.

**Comment by ryujumoto** [comment]
Refer [this code](https://github.com/okuda-seminar/leetcode_answers/blob/main/242_valid_anagram.py).

**Comment by Rwatana** [comment]
I revised. Please check my code.

File: 242_valid_anagram.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# time: O(nlogn)
+# space: O(n)
```
Comment (ryujumoto) [inline 242_valid_anagram.py]: n = len(s)
m = len(t)
time: O(m+n)
space: O(n)

**Review by ryujumoto — COMMENTED** [review]
Check my review.

**Comment by Rwatana** [comment]
I revised. Please check.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**